terraform/dev/stela_deployment.tf

```diff
@@ -0,0 +1,74 @@
+resource "kubernetes_deployment" "stela" {
+  metadata {
+    name = "stela"
+    labels = {
+      App = "stela"
+    }
+  }
+
+  spec {
+    replicas = 2
+    selector {
+      match_labels = {
+        App = "stela"
+      }
+    }
+    template {
+      metadata {
+        labels = {
+          App = "stela"
+        }
+      }
+      spec {
+        container {
+          image = var.stela_image
+          name  = "stela"
+
+          env {
+            name = "DATABASE_URL"
+            value_from {
+              secret_key_ref {
+                name     = "dev-secrets"
+                key      = "DATABASE_URL"
+                optional = false
+              }
+            }
+          }
+
+
+          port {
+            container_port = 80
+          }
+
+          resources {
+            limits = {
+              cpu    = "1"
+              memory = "512Mi"
+            }
+            requests = {
+              cpu    = "250m"
+              memory = "50Mi"
+            }
+          }
```

Comment:
Similar to instance size, we'll probably want to increase the resources here in the not-too-distant future